code/modules/sprite_accessories/_accessory_marking.dm

```diff
@@ -483,25 +483,25 @@
 		name = "Modular Longsock (Right Arm)"
 		icon_state = "osocks"
 		blend = ICON_MULTIPLY
-		body_parts = list(BP_R_ARM)
+		body_parts = list(BP_R_ARM, BP_R_HAND)
```

Comment:
Have we tried to look into the idea of consolidating the arm-hand into the single arm for sprite accessories too? Or would that be too much work for little impact?

Reply:
It would involve a lot of sprite work, as many of the existing bodymarkings were simply designed for codebases with hands and feet. If we really want to consolidate them all, it would be a long term project involving lots of .dmi edits. In the meantime, these bandaid code based fixes will have to suffice.